tests/spaces/test_box.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import pytest
+
+from gym.spaces import Box
+
+# Todo, move Box unique tests from test_spaces.py to test_box.py
+
+
+@pytest.mark.parametrize(
+    "box,expected_shape",
+    [
+        (Box(low=np.zeros(2), high=np.ones(2)), (2,)),
+        (Box(low=np.zeros((2, 1)), high=np.ones((2, 1))), (2, 1)),
+        (Box(low=0, high=1, shape=(5, 2)), (5, 2)),
+        (Box(low=0, high=1), (1,)),
```

Comment:
As a consequence of the previous comment, add a test case where bounds are floats (and maybe (int, float), (float, int) combinations? it's a cheap test so go crazy)

Reply:
Your right, I will add a couple of tests